components/brave_wallet_ui/components/desktop/views/portfolio/index.tsx

```diff
@@ -241,9 +249,10 @@ const Portfolio = (props: Props) => {
       <LineChart
         isDown={selectedAsset && selectedUSDAssetPrice ? Number(selectedUSDAssetPrice.asset24hChange) < 0 : false}
         isAsset={!!selectedAsset}
-        priceData={selectedAsset ? selectedAssetPriceHistory : portfolioPriceHistory}
+        priceData={selectedAsset ? selectedAssetPriceHistory : portfolioHistory}
         onUpdateBalance={onUpdateBalance}
-        isLoading={isLoading}
+        isLoading={selectedAsset ? isLoading : portfolioBalance === '0.00' ? false : isFetchingPortfolioPriceHistory}
+        isDisabled={selectedAsset ? false : portfolioBalance === '0.00'}
```

Comment:
Maybe something like this for the comparison with `0`.

`parseFloat(portfolioBalance) === 0`

Reply:
Done!